src/presets/ec-core/src/ec-core.scss

```diff
@@ -1,2 +1,5 @@
 @use '@ecl/theme-ec-core';
 @use '@ecl/preset-dev/src/dev';
+
+// EC specific (temporary solution)
```

Comment:
I find this solution to be good enough for a standard one. In this case, `footer-core-ec.scss` and `footer-core-eu.scss` are too different to try to accommodate them on the dev layer.

Reply:
Yes, I think that we will have indeed 2 scss files here.
My concern is that we add these files after the others, and not in the order specified in dev preset.
For the footer that's not an issue, but we will have to be careful about it if we do the same for other components

Reply:
Yes, of course we'll be careful with order. I'm very skilled at messing this :)
Yet, let's remove the "(temporary solution)" part as it won't be temporary.
So far, we've had issues in order with icons and links which is not a great amount of code to keep in mind. If we did end up having issues with order, we can split problematic modules out of the dev preset and move them here. Still, I really hope this will be an exception.